lib/foreman_ansible/version.rb

```diff
@@ -4,5 +4,5 @@
 # This way other parts of Foreman can just call ForemanAnsible::VERSION
 # and detect what version the plugin is running.
 module ForemanAnsible
-  VERSION = '9.0.1'
+  VERSION = '10.0.0'
```

Comment:
@Ron-Lavi Shouldn't we change the version just before releasing it?

Reply:
imho it will be more organized to do it once we touch the required foreman version

Reply:
Yop, I'm okay with that if we keep the change in separate commit, so it's easily found in the log